.github/setup-eventbrite.md

```diff
@@ -0,0 +1,124 @@
+| Event Title         | Git & GitHub through GitKraken - from Zero to Hero!       |
+| Organiser           | RSE Sheffield                                             |
+| Type                | Class, Training, or Workshop                              |
+| Category            | Science & Technology                                      |
+| Tags                | git, github, gitkraken                                    |
+| Location            | Online Event (unless otherwise)                           |
+| Date and Time       | Single Event if running over one day, otherwise Recurring |
+| Event Starts        | Start Date/Time (always Single Event), start is usually 09:30 |
+| Event Ends          | End Date/Time if one day finish is usually 16:30, if two 13:00 the following day.  |
+| Time Zone           | United Kingdom Time                                       |
+| Event Page Language | English (UK)                                              |
+
+
+
+## Schedule
+
+If the course is split over two days then you should still have selected **Single Event** as selecting repeating events
+results in people being able to register for individual sessions rather than signing up once for both. This is a current
+short-coming of Eventbrite.
+
+## Details
+
+A banner can be added to the event, some sample images are under the `resources/` directory of this repository, but if
+you have copied a past event this will already be in place.
+
+The following fields should be entered into each field on the **Details** page (they can be copy and pasted from a past
+event).
+
+
+### Summary
+
+
+> This is an intermediate course, teaching the use of Conda virtual environments for reproducible research environments.
+
+### Description
+
+> How can we undertake research work in a consistent and reproducible computing environment?
+> This is an intermediate course, teaching the use of Conda virtual environments for reproducible research envuironments
+> when using Python for research and data analysis. These skills are essential for collaborative research teams.
+> This event is for University of Sheffield researchers and research students only. Please register with an @shef.ac.uk
+> or @sheffield.ac.uk email address to ensure your ticket request is accepted.
+>
+> **Prerequisite skills**:
+> - Basic shell commands (bash/zsh (Linux/MacOS); Powershell (Windows)).
+> - Some Python programming experience.
+>
+> ### Learning Objectives
+> - Understand what Conda is and why it is useful for managing packages and environments in research workflows.
+> - How to create, activate and delete Conda environments.
+> - How to install and manage packages in Conda environments.
+> - What Conda channels and packages are and how to use them.
+> - How to use pip in Conda environments.
+> - How to share your Conda environment.
+> - How to manage packages that have GPU dependencies.
+>
+> This course will be run online.
```

Comment:
Will it?

Reply:
I thought as it might be I'd leave the template as is, its currently how the Git course is configured and its down to the person scheduling to tweak this and the emails appropriately (could have two sets of templates but felt like overkill).

Reply:
Removed, its simple to do the reverse and add a comment that it will be online when we do.